generated_files

```diff
@@ -44,6 +44,9 @@ mixer/template/sample/template.gen.go
 mixer/template/template.gen.go
 mixer/template/tracespan/go_default_library_handler.gen.go
 mixer/template/tracespan/go_default_library_tmpl.pb.go
+mixer/test/spyAdapter/template/report/go_default_library_handler.gen.go
+mixer/test/spyAdapter/template/report/go_default_library_tmpl.pb.go
+mixer/test/spyAdapter/template/template.gen.go
 mixer/test/template/report/go_default_library_handler.gen.go
```

Comment:
Bottom 3 files must be gone now. I think

Reply:
@jmuk how is this file updates ? Any idea why this change was missed when @geeknoid submitted the PR that moved the .gen.go files ?